<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="FN">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FILE NAME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rec">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuRecording">
    <property name="title">
     <string>Recording</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionStart_2"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionPause"/>
    <addaction name="actionPlay"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRecording"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionStart_2">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
